process/publish.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e
+
+# Clone the ar-io-network-process repo and copy src directory
+if [ "$1" = "--reset" ]; then
+  echo "Refreshing directory..."
+  rm -rf ./src
+  echo "Cloning ar-io-network-process repo..."
+  # TODO: drop in a final hash of the network process source code
+  git clone -b develop https://github.com/ar-io/ar-io-network-process.git tmp-ar-io
+  mkdir -p ./src
+  cp -r tmp-ar-io/src/* ./src/
+  rm -rf tmp-ar-io
+
+  # TODO: drop in a final hash of the csvs and lua from this repo in place of develop branch
+  echo "Fetching finalized state from mainnet-csv repo..."
+  mkdir -p ./state
+  git clone https://github.com/ar-io/ar-io-mainnet-csvs tmp-mainnet-csv
+  cp -r tmp-mainnet-csv/state/* ./state/
+  rm -rf tmp-mainnet-csv
+  shift # remove --reset from arguments
```

Comment:
I guess this implies that argument ordering matters for this script's invocation?

Reply:
correct